EightMinuteEmpireLegends/Core/Map/Map.cpp

```diff
@@ -8,24 +8,30 @@
 Map::Map(std::string name, std::vector<Continent*> c)
 	: name(name), v_vertices({}), v_edges({}), v_continents(c), startingRegion(nullptr) { }
 
+Map::Map(Map* m)
```

Comment:
I think for copy constructor you should have the parameter as const so that it cant be changed. I think this is the correct signature: ClassName (const ClassName &old_obj); Also make sure that you are making a deep (not shallow) copy of the member objects.